static/locales/fr/translation.json

```
@@ -7,26 +7,26 @@
 			"title": "Paramètres du lobby",
 			"gamehostonly": "Seul l'hôte de la partie peut modifier cela!",
 			"inlobbyonly": "Vous ne pouvez changer cela que dans le lobby!",
-			"voicedistance": "Distance vocal",
-			"voicedistance_impostor": "Distance vocal imposteurs",
+			"voicedistance": "Distance vocale",
+			"voicedistance_impostor": "Distance vocale imposteurs",
 			"wallsblockaudio": "Les murs bloquent l'audio",
-			"visiononly": "Entendre uniquement les gens que l'on voit",
+			"visiononly": "Entendre uniquement les gens visibles activé",
 			"impostorshearsghost": "Les imposteurs entendent les fantômes",
 			"hear_imposters_invents": "Entendre les imposteurs dans les vents",
 			"private_talk_invents": "Discussion privée dans les vents",
 			"comms_sabotage_audio": "Saboter les communications désactive le vocal",
 			"hear_through_cameras": "Entendre à travers les caméras",
 			"ghost_only": "Seuls les fantômes peuvent parler/entendre",
 			"ghost_only_warning": "Cette option désactive le vocal pour les joueurs vivants.",
-			"ghost_only_warning2": "Seuls les fantômes peuvent parler activé.",
+			"ghost_only_warning2": "Seuls les fantômes peuvent parler activé",
```

```suggestion
			"ghost_only_warning2": "Seuls les fantômes peuvent parler",
```
If I understand it correctly, this message will be displayed in the text on the left of the lobby only if the option is activated? In this case, there is no need to keep the "enabled" part, especially since it makes the sentence quite difficult to understand.

---

Yeah, the "enabled" is more to say that this option is enabled but I think everyone will know what it means even if there is no "enabled"